server/ai_http.go

```diff
@@ -163,6 +164,8 @@ func (h *lphttp) StartLiveVideoToVideo() http.Handler {
 		subCh.CreateChannel()
 		controlPubCh := trickle.NewLocalPublisher(h.trickleSrv, mid+"-control", "application/json")
 		controlPubCh.CreateChannel()
+		eventsCh := trickle.NewLocalPublisher(h.trickleSrv, mid+"-events", "application/json")
```

Comment:
Same, thought this was supposed to be a subscriber. Or is this orchestrator code?